
#Change for sim or hardware experiment
is_sim: True

#Number of agents that are going to be used in the experiment
number_of_agents: 6


#Bounding Box

bbox_x: 0.8
bbox_y: 0.8
bbox_z: 0.8



